test/functional/eval/backtick_expansion_spec.lua

```diff
@@ -21,11 +21,19 @@ describe("backtick expansion", function()
   end)
 
   it("with default 'shell'", function()
-    if helpers.pending_win32(pending) then return end  -- Need win32 shell fixes
-    command(":silent args `echo ***2`")
+    if helpers.os_name() == 'windows' then
```

Comment:
`helpers.iswin()` is preferred because it's faster, doesn't require `clear()`, and is easier to read.